governance-server/src/logic.js

```diff
@@ -271,6 +282,37 @@ module.exports = class Logic {
         return discoverDelegationChain(address);
     };
 
+  /**
+   * Creates a delegation tree downwards to see who this account had delegations to
+   * @param address
+   * @param height
+   * @param ignoreAccounts
+   * @returns {Promise<*>}
+   */
+  flatDelegateeTree = async (address, height, ignoreAccounts = []) => {
```

Comment:
the below function `delegatedPower` also has logic to flatten the delegation tree, I guess we should try to not have duplicate logic here.